### PR TITLE
Fix almost all tests for ie11

### PR DESCRIPTION
**Pull request by cibernox**
I think there is still 2 failing tests, but I'm out for the day.

### CODE REVIEW

File: addon-test-support/@ember/test-helpers/dom/fill-in.js [hunk 1]
```diff
@@ -22,8 +22,8 @@ export default function fillIn(target, text) {
     if (!element) {
       throw new Error(`Element not found when calling \`fillIn('${target}')\`.`);
     }
-
-    if (!isFormControl(element) && !element.isContentEditable) {
+    let isControl = isFormControl(element);
```
Comment (cibernox) [inline addon-test-support/@ember/test-helpers/dom/fill-in.js]: It turns out that `isContentEditable` is true for `input` elements in IE. I did this fix but maybe we have to go back to check if the element has a `contenteditable=true` attribute

File: tests/dummy/config/targets.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /* eslint-env node */
-const browsers = ['last 1 Chrome versions', 'last 1 Firefox versions', 'last 1 Safari versions'];
+const browsers = ['ie 11', 'last 1 Chrome versions', 'last 1 Firefox versions', 'last 1 Safari versions'];
```
Comment (rwjblue) [inline tests/dummy/config/targets.js]: We should probably revert this change (to test locally you can use `CI=true ember test --server` or `CI=true ember s -p 0`)...

File: tests/helpers/is-ie.js [hunk 3]
```diff
@@ -0,0 +1 @@
+export default !window.ActiveXObject && 'ActiveXObject' in window;
```
Comment (rwjblue) [inline tests/helpers/is-ie.js]: This is great! 👍  I think we should add a comment explaining what it does.

Something like:

```js
// `window.ActiveXObject` returns undefined in IE11 (as well as non-IE browsers)
// whereas `"ActiveXObject" in window` returns `true` in all IE versions
// only IE11 will pass _both_ of these conditions
```

Comment (ro0gr) [reply tests/helpers/is-ie.js]: 😨

**Comment by rwjblue** [comment]
OK, pushed some changes / tweaks to address my comments above.

IE11 PASSING

**Comment by cibernox** [comment]
Now great. Now I need an IE10 vm

**Comment by rwjblue** [comment]
Great work here @cibernox! Thank you!

**Pull request merged**